liquibase-core/src/main/java/liquibase/parser/core/java/JavaChangeLogParser.java

```diff
@@ -0,0 +1,61 @@
+package liquibase.parser.core.java;
+
+import liquibase.Scope;
+import liquibase.change.Change;
+import liquibase.changelog.ChangeLogParameters;
+import liquibase.changelog.DatabaseChangeLog;
+import liquibase.exception.ChangeLogParseException;
+import liquibase.parser.ChangeLogParser;
+import liquibase.resource.ResourceAccessor;
+
+import java.lang.reflect.Method;
+
+public class JavaChangeLogParser implements ChangeLogParser {
+
+    @Override
+    public DatabaseChangeLog parse(String physicalChangeLogLocation, ChangeLogParameters changeLogParameters, ResourceAccessor resourceAccessor) throws ChangeLogParseException {
+        try {
+            Class changeLogClass = Class.forName(physicalChangeLogLocation.replace("/", "."));
+            Object changeLogInstance = changeLogClass.newInstance();
+
+            DatabaseChangeLog changelog = new DatabaseChangeLog(physicalChangeLogLocation);
+
+            for (Method method : changeLogClass.getMethods()) {
```

Comment:
I didn't bother to sort the methods by the order annotation attribute yet. I saw you were using a reflections library in your code, but I like to keep the 3rd party libraries to a minimum to better fit into whatever environment people have so if we can do without that it would be nice.